### PR TITLE
Add Example.java and adjust README

### PR DESCRIPTION
**Pull request by FatlumL**

### CODE REVIEW

**Comment by FatlumL** [comment]
Adding Test.java as well

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# myfirstproject
+# My First Project
```
Comment (fgogolli) [inline README.md:2]: Remove spaces

File: Test.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This is a test file
+
+public class Test {
+	public static void main(String[] args){
+		# Do something here
+	}
+}
```
Comment (fgogolli) [inline Test.java:7]: Add newline

**Review by fgogolli — CHANGES_REQUESTED** [review]

**Review by fgogolli — APPROVED** [review]
LGTM

**Pull request merged**